app/models.rb

```diff
@@ -3,7 +3,7 @@
 require 'sequel'
 
 ENV['DATABASE_URL'] ||= "postgres://localhost/citygram_#{Citygram::App.environment}"
-DB = Sequel.connect(ENV['DATABASE_URL'])
+DB = Sequel.connect(ENV['DATABASE_URL'], max_connections: 20)
```

Comment:
@fma2 I believe this will fix the timeout issues. I've tested it with staging emails, now we need to test with live.